skaffold/src/main/kotlin/com/google/container/tools/skaffold/run/SkaffoldCommandLineState.kt

```diff
@@ -66,11 +67,13 @@ class SkaffoldCommandLineState(
             configFile, projectBaseDir
         )
 
+        val workingDirectory: File? = File(projectBaseDir.path)
+
         val skaffoldProcess = SkaffoldExecutorService.instance.executeSkaffold(
             SkaffoldExecutorSettings(
                 executionMode,
                 skaffoldConfigurationFilePath,
-                workingDirectory = File(environment.project.basePath),
+                workingDirectory = workingDirectory,
```

Comment:
why not just keep it inline instead of defining `workingDIrectory`?

Reply:
Agree, remainder of refactoring